airflow_dags/dag.py

```diff
@@ -166,3 +166,17 @@ def get_args_helper(loader_builder: DoltLoaderBuilder, remote_url: str):
                                                             WIKIPEDIA_NGRAMS_REPO),
                                   dag=wikipedia_ngrams_dag)
 
+# Backfill Wikipedia ngrams
+wikipedia_ngrams_backfill_dag = DAG(
+    'wikipedia-ngrams-backfill',
+    default_args=get_default_args_helper(datetime(2019, 12, 2)),
+    schedule_interval='@once',
+)
+
+dump_dates = ['20190820', '20190901', '20190920', '20191001', '20191020', '20191101', '20191120']
```

Comment:
@timsehn this was easier than using `catchup` and creating logic in `dolt_load` since there are only a few dumps and I know exactly which ones I need